lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -999,6 +999,14 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => false,
           :description  => 'Controls auto-instrumentation of dalli gem for Memcache at start up.  May be one of [auto|prepend|chain|disabled].'
         },
+        :'instrumentation.logger' => {
+          :default => instrumentation_value_of(:disable_logger_instrumentation),
```

Comment:
"auto"

Reply:
🆒  📚

Reply:
Added in https://github.com/newrelic/newrelic-ruby-agent/pull/798/commits/2a9cb148da9365e9aa08331338d52deff570ba65